src/main/scala/org/ergoplatform/nodeView/wallet/WalletScanLogic.scala

```diff
@@ -65,6 +65,11 @@ object WalletScanLogic extends ScorexLogging {
                             blockId: ModifierId,
                             transactions: Seq[ErgoTransaction]): (WalletRegistry, OffChainRegistry) = {
 
+    //input tx id, input box id, tracked box
+    type InputData = Seq[(ModifierId, EncodedBoxId, TrackedBox)]
+    //outputs, input ids, related transactions
+    type ScanResults = (Seq[TrackedBox], InputData, Seq[WalletTransaction])
```

Comment:
I suggest to convert tuples into case classes to improve readability.

Reply:
reverted the class instead, modifying it is out of scope of the PR.